### DISCUSSION TITLE
Is Apache 2.0 license correct?

### DISCUSSION THREAD

**Discussion by schlomo** [open-gitops/project#81] [Q&A] [answered]
I noticed that the license here is Apache 2.0 - is this really appropriate for a repo that won't provide code but rather documentation? Maybe CC-BY-SA would be better suited?

**Comment by tonit** [comment]
Question to @cdavisafc and generally current maintainers. It looks like for "demo code" ASL2 is already used elsewhere (looking at the very good potatohead project of the app delivery sig).

**Comment by scottrigby** [comment]
Precedence:

Both Apache 2.0 and CC by 4.0:
- https://github.com/cncf/cnf-wg/blob/master/LICENSE.md

Apache 2.0 only:
- https://github.com/cncf/wg-ci/blob/master/LICENSE
- https://github.com/cncf/wg-serverless/blob/master/LICENSE
- https://github.com/cncf/wg-networking/blob/master/LICENSE
- https://github.com/cncf/udpa/blob/main/LICENSE

**Reply by scottrigby** [reply]
@schlomo I've converted this issue to a discussion. Does this answer your question?

**Reply by scottrigby** [reply]
@schlomo I have referenced this discussion from https://github.com/open-gitops/principles/issues/2

**Comment by scottrigby** [comment, marked as answer]
Answering this for posterity. The GItOps Working Group (this repo) has adopted Apache 2 ([local copy](https://github.com/gitops-working-group/gitops-working-group/blob/main/LICENSE)), and we have since decided to adopt both licenses for OpenGitOps projects: Creative Commons for content, Apache 2 for any code.

See https://github.com/open-gitops/project/blob/main/LICENSE.md.
